plugins/nimble/nimble.go

```diff
@@ -0,0 +1,27 @@
+package nimble
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+)
+
+func nimCLI() schema.Executable {
+	return schema.Executable{
+		Name:    "nimble",
+		Runs:    []string{"nimble"},
+		DocsURL: sdk.URL("https://github.com/nim-lang/nimble"),
+		NeedsAuth: needsauth.IfAll(
+			needsauth.IfAny(
```

Comment:
No need for `IfAny` in this situation, since it only contains one rule.

Reply:
This has yet to be resolved